docs/source/development/start/support.rst

```diff
@@ -0,0 +1,8 @@
+Getting Help
```

Comment:
Rename this file to `help.rst`? since community support is another concept in a different folder that is linked to from here.

Reply:
Good point, taken.